lessons/ro/chapter_1.yaml

```
@@ -0,0 +1,227 @@
+  content_markdown: >
+    O funcție admite mai multe argumente sau deloc.
+
+
+    În acest exemplu, funcția *add* admite două argumente de tip `i32` 
+    (signed integer cu dimensiune de 32 de biți).
+
+
+    Dacă vreți să returnați o valoare sau expresie, puteți folosi cuvântul cheie `return`
+    și punct și virgula la final, așa cum am făcut în funcția *subtract*.
+
+
+    Numele funcțiilor sunt mereu scrise în format `snake_case`.
+
+
+    Hint: dacă definiți o funcție, datele pe care le admite se numesc parametrii.
+    Dacă folosiți o funcție și îi transmiteți niște date, atunci ele se numesc argumente.
+- title: Returnarea mai multor valori
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=fn%20swap(x%3A%20i32%2C%20y%3A%20i32)%20-%3E%20(i32%2C%20i32)%20%7B%0A%20%20%20%20return%20(y%2C%20x)%3B%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20%2F%2F%20return%20a%20tuple%20of%20return%20values%0A%20%20%20%20let%20result%20%3D%20swap(123%2C%20321)%3B%0A%20%20%20%20println!(%22%7B%7D%20%7B%7D%22%2C%20result.0%2C%20result.1)%3B%0A%0A%20%20%20%20%2F%2F%20destructure%20the%20tuple%20into%20two%20variables%20names%0A%20%20%20%20let%20(a%2C%20b)%20%3D%20swap(result.0%2C%20result.1)%3B%0A%20%20%20%20println!(%22%7B%7D%20%7B%7D%22%2C%20a%2C%20b)%3B%0A%7D%0A
+  content_markdown: >
+    Funcțiile pot returna mai multe valori prin returnarea unui **tuplu** (tuple)
+
+    de valori.
+
+
+    Elementele unui tuplu pot fi accesate folosind indexul acestora. (ex: my_tuple.0)
+
+
+    Rust suportă diferite tipuri de destructurări pe care le vom vedea în multe contexte,
+    permințându-ne
+
+    să extragem fracțiuni ale unor structuri de date în moduri ingenioase. Fiți pe fază!
+- title: Returnarea
+  content_markdown: >
+    Dacă pentru o funcție nu se specifică ce tip returnează, aceasta va returna un tuplu gol,
+    cunoscut și sub numele de *unitate* (unit).
+
+
+    Un tuplu gol este reprezentat de `()`.
+
+
+    Folosirea unui `()` nu este des întâlnită, dar va apărea de suficiente ori, deci este bine să știți
+    ce se întâmplă.
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=fn%20make_nothing()%20-%3E%20()%20%7B%0A%20%20%20%20return%20()%3B%0A%7D%0A%0A%2F%2F%20the%20return%20type%20is%20implied%20as%20()%0Afn%20make_nothing2()%20%7B%0A%20%20%20%20%2F%2F%20this%20function%20will%20return%20()%20if%20nothing%20is%20specified%20to%20return%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20let%20a%20%3D%20make_nothing()%3B%0A%20%20%20%20let%20b%20%3D%20make_nothing2()%3B%0A%0A%20%20%20%20%2F%2F%20Printing%20a%20debug%20string%20for%20a%20and%20b%0A%20%20%20%20%2F%2F%20Because%20it's%20hard%20to%20print%20nothingness%0A%20%20%20%20println!(%22The%20value%20of%20a%3A%20%7B%3A%3F%7D%22%2C%20a)%3B%0A%20%20%20%20println!(%22The%20value%20of%20b%3A%20%7B%3A%3F%7D%22%2C%20b)%3B%0A%7D%0A
```

```suggestion
    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=fn%20make_nothing()%20-%3E%20()%20%7B%0A%20%20%20%20return%20()%3B%0A%7D%0A%0A%2F%2F%20the%20return%20type%20is%20implied%20as%20()%0Afn%20make_nothing2()%20%7B%0A%20%20%20%20%2F%2F%20this%20function%20will%20return%20()%20if%20nothing%20is%20specified%20to%20return%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20let%20a%20%3D%20make_nothing()%3B%0A%20%20%20%20let%20b%20%3D%20make_nothing2()%3B%0A%0A%20%20%20%20%2F%2F%20Printing%20a%20debug%20string%20for%20a%20and%20b%0A%20%20%20%20%2F%2F%20Because%20it's%20hard%20to%20print%20nothingness%0A%20%20%20%20println!(%22The%20value%20of%20a%3A%20%7B%3A%3F%7D%22%2C%20a)%3B%0A%20%20%20%20println!(%22The%20value%20of%20b%3A%20%7B%3A%3F%7D%22%2C%20b)%3B%0A%7D%0A
```
Comentariile trebuie traduse